composer.json

```diff
@@ -19,6 +19,11 @@
     "autoload": {
         "psr-4": {
             "Develpr\\AlexaApp\\": "src"
-           }
+        }
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "0.4-dev"
```

Comment:
@develpr When you release a new `0.x.0` version, bump this to the next-highest one.

For example, just before you release `0.4.0`, change this value to `"0.5-dev"`.  That'll tell Composer to consider `master` as the development branch of `0.5.0`.  (You can keep releasing `0.4.x` versions while this is set to `0.5-dev` - that's totally fine.)